src/redux/actions/experiments/createExperiment.js

```diff
@@ -77,6 +64,8 @@ const createExperiment = (
         error: errorMessage,
       },
     });
+
+    throw e;
```

Comment:
Why throw here? Significant error that we want to crash and notify us on slack instead of only showing a notification message to the user?

Reply:
I added the `throw` because `createExperiment` is only called in `createProject`, I also set `createProject` to dispatch an error and not create the project if `createExperiment` fails.

It will be simplified (and we'll remove this `throw`) when we remove the concept of `project` and then we can get rid of `createProject` altogether.